spec/support/ansible_shared/automation_manager/refresher.rb

```diff
@@ -100,12 +100,12 @@
   def assert_counts
     expect(Provider.count).to                                 eq(1)
     expect(automation_manager).to                             have_attributes(:api_version => "3.0.1")
-    expect(automation_manager.configured_systems.count).to    eq(130)
```

Comment:
I did and ended up aborting it because it needs enterprise license to support survey spec.